README.md

```diff
@@ -65,6 +65,31 @@ Currently the waiting period is as follows:
 
 ## FAQ
 
+#### Do I need to register a package to install it?
```

Comment:
Content looks great, but I think the distinction between the two questions is so blurred that they could be merged into a single one

Reply:
Yeah, that's true. I wrote it like that at first actually and then split it up, since I thought that followed the style of the FAQ better. I think either way is OK though. If we add more to this one (like Lyndon's sugestions) then I have a preference for keeping them separate so that we don't have one giant entry.